### ISSUE TITLE
Works fine just 1 thing...

### ISSUE THREAD

**Issue by SephVII** [suzaku01/mhf_text_generator#1]
Worked fine for me so far just one thing, is it possible to limit the Box where you write in the text to generate the code to the character limit the Box has ingame ? otherwise you can overflow the text over the borders easly.

**Comment by suzaku01** [comment]
Use linebreak in textbox.

**Comment by SephVII** [comment]
> Use linebreak in textbox.

Yeah i do use linebreak. I meant if you write the text you need to save the generated text to the .go file then check ingame if it does overflow out of the box. And then go back and fix it and do the whole process again until its done.
